src/components/show-products/index.js

```diff
@@ -16,9 +16,33 @@ export const ShowProducts = () => {
   const { products, loading } = useProducts();
   const { push } = useHistory();
   const { storedValue } = useLocalStorage(LOCAL_STORAGE_LIST_TOKEN);
-  const timeFrames = TimeFrames;
+  const [searchTerm, setSearchTerm] = useState('');
+  const [list, setList] = useState([]);
 
-  const handleOnChange = (productID) => {
+  useEffect(() => {
+    let results = products;
+
+    if (searchTerm !== '') {
+      results = products.filter(({ productName }) =>
+        productName
+          .toLowerCase()
+          .replace(/\s/g, '')
+          .normalize('NFD')
+          .replace(/[\u0300-\u036f]/g, '')
+          .includes(
+            searchTerm
+              .toLowerCase()
+              .replace(/\s/g, '')
+              .normalize('NFD')
+              .replace(/[\u0300-\u036f]/g, ''),
+          ),
+      );
+    }
+
+    setList(results);
```

Comment:
Here, if there is no results, it would be quite nice to return something so the user's able to know that there is no matches, especially for screen readers.

Reply:
Done

Reply:
E X C E L L E N T !!! :100: